book/src/assets.md

```diff
@@ -0,0 +1,13 @@
+# Assets
+
+**Assets** are configuration that are loaded by a game when it is run. These may be textures, sounds, game level scripts, and so on. In fact, any non-player-specific information loaded by the executable can be called an asset. These are usually stored as files, and distributed alongside the game.
```

Comment:
"In fact, any non-player-specific information loaded by the executable can be called an asset." it is unclear here what "non-player-specific information" is, my first thought for this sentence would maybe be "...any data that is loaded at runtime may be considered an asset."

Reply:
+1

Reply:
I can roll with that; was trying to disambiguate "information that is common to all users" from "information that changes per user".